### PR TITLE
Font: unify font styling and KeyValuePage: improve looks

### LINKED ISSUE #2578: FR: nicer keyvaluepage
Copied from https://github.com/koreader/koreader/issues/2562#issuecomment-282469008

I'd like this:

screenshot_2017-02-25_09-16-33

To go more in this direction (Evince properties dialog):

screenshot_2017-02-25_09-16-17

### PR DESCRIPTION
**Pull request by Frenzie**
Fixes #2578.

* key bolded
* values normally left-aligned at 50%
* allows misalignment for the sake of fitting everything on one line

## Before
screenshot_2017-04-25_20-02-53

## After
screenshot_2017-04-25_20-01-48

## Before
screenshot_2017-04-25_20-03-00

## After
screenshot_2017-04-25_20-02-08

## Before
screenshot_2017-04-25_20-08-34

## After
screenshot_2017-04-25_20-08-58

## Overflow
screenshot_2017-04-25_20-02-33

This change is thanks to @houqp, because otherwise I'd have spent my time on #2809 instead. ;-)

### CODE REVIEW

**Comment by poire-z** [comment]
Couldn't we also decrease font size, and add some small margin to the page ?
(I actually like how the book status page looks, it's sharp and clean, compared to the original keyvaluepage.)

**Comment by Frenzie** [comment]
I'm personally fine with that but do note #2563.

**Comment by poire-z** [comment]
Yes indeed (dunno how that book status looks on kobo mini, if it's unreadable or if there's there some font adjusment depending on device screen).

**Comment by Frenzie** [comment]
It did occur to me that I ever so slightly mismatched the font sizes and didn't even notice (I thought it was because it was bold).

From font.lua:
```lua
        cfont = 24,
        tfont = 26,
```

The size @baskerville used was `21`, so I've tried `22` instead. On the screenshot implementing your suggestion looks a *lot* better, but on anything with a bigger screen it should look fairly similar even with size 24 and some padding.

screenshot_2017-04-26_14-55-52
screenshot_2017-04-26_14-56-01
screenshot_2017-04-26_14-56-08

File: frontend/ui/widget/keyvaluepage.lua [hunk 1]
```diff
@@ -113,7 +113,8 @@ end
 local KeyValueItem = InputContainer:new{
     key = nil,
     value = nil,
-    cface = Font:getFace("cfont"),
+    cface = Font:getFace("cfont", 22),
```
Comment (houqp) [inline frontend/ui/widget/keyvaluepage.lua]: Font size should be wrapped by `Screen:scaleBySize`. And I suggest changing the default size in `Font` module so we get a consistent experience across the app.

**Comment by houqp** [comment]
Two nitpicks, the rest looks good to me. It looks much better after the patch :+1:

File: frontend/ui/widget/keyvaluepage.lua (hunk 1 above)
Comment (Frenzie) [inline frontend/ui/widget/keyvaluepage.lua]: It shouldn't; that's done by getFace and if it weren't that's still where it should be done. ;-)

File: frontend/ui/widget/keyvaluepage.lua [hunk 2]
```diff
@@ -130,36 +131,56 @@ function KeyValueItem:init()
...
     end
 
-    local key_w = RenderText:sizeUtf8Text(0, self.width, self.cface, self.key).x
-    local value_w = RenderText:sizeUtf8Text(0, self.width, self.cface, self.value).x
-    if key_w + value_w > self.width then
-        -- truncate key or value so they fits in one row
-        if key_w >= value_w then
-            self.show_key = RenderText:truncateTextByWidth(self.key, self.cface, self.width-value_w)
-            self.show_value = self.value
+    local frame_padding = 8
```
Comment (Frenzie) [inline frontend/ui/widget/keyvaluepage.lua]: From @poire-z's suggestion to add padding? That is, just from playing around with a few values.

Comment (houqp) [inline frontend/ui/widget/keyvaluepage.lua]: i see. We should use `Screen:scaleBySize` on raw numbers to make it consistent on different screen sizes.

File: frontend/ui/widget/keyvaluepage.lua (hunk 1 above)
Comment (houqp) [inline frontend/ui/widget/keyvaluepage.lua]: Ha, good catch, I forgot Font module already does this internally ;)

**Pull request merged**